pkgs/development/php-packages/relay/default.nix

```
@@ -0,0 +1,99 @@
+{ stdenv
+, lib
+, php
+, openssl
+, openssl_1_1
+, zstd
+, lz4
+, autoPatchelfHook
+}:
+
+let
+  version = "0.6.3";
+  system = stdenv.hostPlatform.system;
+  phpVersion = lib.versions.majorMinor php.version;
+  variation = {
+    "aarch64-darwin" = {
+      platform = "darwin-arm64";
+      hashes = {
+        "8.0" = "0g2zg266kha741yzy18ys4rbky92d4dl4vr1xzpc9dg1j6in5pqw";
+        "8.1" = "0diyvcnfm1csgy3072xf24zscaf8vg6jy5bb604i2ihkvfpjcf62";
+        "8.2" = "0s2swrmmkc0raasv5bp970pwyn6n570jnm3nxlyqg3hyzyyibms7";
+        "8.3" = "0ka3q711ni4x1pgxbf6ccv0bg5zla078z0q464f49f8fnlvgrmvj";
+      };
+    };
+    "aarch64-linux" = {
+      platform = "debian-aarch64+libssl3";
+      hashes = {
+        "8.0" = "1by11rz1w4lv9hwvcwdgg56swy2nhrrhfa4f6zqvljzygwlcizqj";
+        "8.1" = "1fc7x09l1al489pn0lpbfb402dyz8p9jjiif3k3ppggb7ns0zp6z";
+        "8.2" = "1ln7mh2kps713a4psvqsfwxygn4mxgcqf8bjg44x0mdwg62n454x";
+        "8.3" = "1nnqr4w9zbgi1phwzv15lgqn13c2fi5qkyx9ayib1l3hm371p1vw";
+      };
+    };
+    "x86_64-darwin" = {
+      platform = "darwin-x86-64";
+      hashes = {
+        "8.0" = "0bqgb8wlfjpx9pdh2arx7dhlrx6mfnyid0kf8jyv48afqcgbjqny";
+        "8.1" = "0mj663ghwq0zkiqbjgb4g08jcijjz24c055rns36b5kgaf4sjvfd";
+        "8.2" = "1i1bk3yiw3dk8p3yf11g7g90bfyb7r31f58c7bhn6vqkdw6r4prm";
+      };
+    };
+    "x86_64-linux" = {
+      platform = "debian-x86-64+libssl3";
+      hashes = {
+        "8.0" = "1ind8bf2dm5r1f1y10hipskyn484nc967cz63z6d9n4war56g3yb";
+        "8.1" = "1jvrj5vmdmj1vk2gcd6cfg9n21qn71nmyl7q8p20vbg3s9q7kvs9";
+        "8.2" = "1vrhfl6nnzl8vkra2i20gn8ddb95b04dikjmrdp53kmyn2lcnwf4";
+        "8.3" = "0nr4xlaqi3azy0yfx5lkv8ggmvdh4wh1zrycjfpsddz31gyfhi39";
+      };
+    };
+  }.${system} or (throw "Unsupported platform for relay: ${system}");
+in
+stdenv.mkDerivation {
+  inherit version;
+  name = "relay";
+  extensionName = "relay";
+
+  src = builtins.fetchTarball {
+    url = "https://builds.r2.relay.so/v${version}/relay-v${version}-php"
+      + phpVersion + "-" + variation.platform + ".tar.gz";
+    sha256 = variation.hashes.${phpVersion} or (throw "Unsupported PHP version for relay ${phpVersion} on ${system}");
+  };
+  nativeBuildInputs = if stdenv.isDarwin then [ ] else [ autoPatchelfHook ];
+  buildInputs = [ openssl zstd lz4 ];
```

Could you make the list one dependency per line? It will make future diffs nicer to read and `git blame` easier to use.

```suggestion
  nativeBuildInputs = lib.optionals (!stdenv.isDarwin) then [
    autoPatchelfHook
  ];

  buildInputs = [
    openssl
    zstd
    lz4
  ];
```